Source/Renderer/Texture.js

```diff
@@ -347,6 +360,7 @@ function Texture(options) {
   this._sizeInBytes = sizeInBytes;
   this._preMultiplyAlpha = preMultiplyAlpha;
   this._flipY = flipY;
+  this._skipColorSpaceConversion = skipColorSpaceConversion;
```

Comment:
I don't think this should to be saved as a member variable of `Texture`. Instead it should be an option in `Texture` and `Texture.prototype.copyFrom`.

The way I think of it is it's not the texture that inherently says to skip the color conversion, it's an option that's tied to the source data being sent through.